app/controllers/stories/feeds_controller.rb

```diff
@@ -27,23 +27,49 @@ def show
     private
 
     def assign_feed_stories
-      feed = Articles::Feed.new(user: current_user, page: @page, tag: params[:tag])
       stories = if params[:timeframe].in?(Timeframer::FILTER_TIMEFRAMES)
-                  feed.top_articles_by_timeframe(timeframe: params[:timeframe])
+                  timeframe_feed
                 elsif params[:timeframe] == Timeframer::LATEST_TIMEFRAME
-                  feed.latest_feed
+                  latest_feed
                 elsif user_signed_in?
-                  ab_test_user_signed_in_feed(feed)
+                  signed_in_base_feed
                 else
-                  feed.default_home_feed(user_signed_in: user_signed_in?)
+                  signed_out_base_feed
                 end
       ArticleDecorator.decorate_collection(stories)
     end
 
-    def ab_test_user_signed_in_feed(feed)
+    def signed_in_base_feed
+      if SiteConfig.feed_strategy == "basic"
```

Comment:
I'd really prefer if we could do something like this in the long run (can be a follow-up PR):

1. move the top-level methods into the service classes: `Feeds::Basic.signed_in_base_feed`, `Feeds::Optimized.signed_out_base_feed`, etc.
2. Add a method here for picking the correct strategy, e.g. 
    ```ruby
    def strategy
      Feed.const_get(SiteConfig.feed_strategy.classify)
    end
   ```
3. Use the strategies here:
    ```ruby
    def  signed_in_base_feed
      strategy.signed_in_base_feed
    end

    # or even
    delegate :signed_in_base_feed, :signed_out_base_feed, to: :strategy
    ```

This way we can add more strategies without having to touch this controller.